cs/src/Contracts/Tunnel.cs

```diff
@@ -37,8 +37,8 @@ public Tunnel()
     /// Gets or sets the generated ID of the tunnel, unique within the cluster.
     /// </summary>
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-    [RegularExpression(OldTunnelIdPattern)]
-    [StringLength(OldTunnelIdLength, MinimumLength = OldTunnelIdLength)]
+    [RegularExpression(NewTunnelIdPattern)]
+    [StringLength(NewTunnelIdMaxLength, MinimumLength = NewTunnelIdMinLength)]
     public string? TunnelId { get; set; }
```

Comment:
Changing `TunnelV2` to become `Tunnel` will be a breaking change when integrated into service. Old clients with old SDK still are going use old `Tunnel` object, but the service will treating it as `TunnelV2`. Same for `TunnelPortV2`

Reply:
If you want to make new SDK's `Tunnel` be what `TunnelV2` is now, you need to keep old tunnel as `TunnelV1` and then change service to return `TunnelV1` where it now returns `Tunnel`.  Same for tunnel port.

Reply:
I have a draft PR of how that will work. OldTunnel could be just hosted on the service and TunnelV2 and Tunnel will be renamed appropriately. That way we dont have to deal with qualifiers in tunnel types going forward 
https://devdiv.visualstudio.com/OnlineServices/_git/tunnels/pullrequest/500108